### PR TITLE
iFlight BLITZ Mini F745

### PR DESCRIPTION
**Pull request by andyp1per**
https://shop.iflight.com/BLITZ-Mini-F745-Flight-Controller-Pro2142

### CODE REVIEW

**Comment by Hwurzburg** [comment]
why is the Blitz745 in this PR??...its already in its own...will review only the mini portion

**Comment by andyp1per** [comment]
> why is the Blitz745 in this PR??...its already in its own...will review only the mini portion

Because one is based on the other

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
All the readme changes I suggested for the BlitzF745 should be made on this one also

File: libraries/AP_HAL_ChibiOS/hwdef/BlitzF745Mini/defaults.parm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# setup for LEDs on chan9
```
[the remark this answers is not in the document]
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/BlitzF745Mini/defaults.parm]: Fixed

**Review by Hwurzburg — APPROVED** [review]
the change to base will need to be added....cant you kill the BlitzF745 and just change the commit title to both boards? or is that not allowed....normally I just make sure the PRs are separate and need to be merged in order

**Comment by rmackay9** [comment]
This is included in 4.5.2-beta1

**Pull request merged**